<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPdlg</class>
 <widget class="QDialog" name="UDPdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPdlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>T&amp;ext : </string>
         </property>
         <property name="buddy">
          <cstring>lineEditTx</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTx">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botTx">
         <property name="text">
          <string>Tr&amp;amet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;IP servidor : </string>
         </property>
         <property name="buddy">
          <cstring>lineEditIP</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditIP">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="inputMask">
          <string>000.000.000.000</string>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port &amp;Tx : </string>
           </property>
           <property name="buddy">
            <cstring>lineEditPortTx</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPortTx">
           <property name="text">
            <string>45454</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="botIntercanvi">
         <property name="text">
          <string>&lt;-&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Port &amp;Rx : </string>
           </property>
           <property name="buddy">
            <cstring>lineEditPortRx</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPortRx">
           <property name="text">
            <string>45455</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditTx</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UDPdlg</receiver>
   <slot>slotMiroSiHiHaText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPortRx</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UDPdlg</receiver>
   <slot>slotCanviaPortRx(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotMiroSiHiHaText(QString)</slot>
  <slot>slotCanviaPortRx(QString)</slot>
 </slots>
</ui>
